src/components/Tab/style.scss

```diff
@@ -40,11 +39,12 @@
     font-weight: 700;
 
     .tab__description {
-      color: $font-primary-color;
+      color: #8e949a;
```

Comment:
any link to design?

Reply:
Sure: https://app.zeplin.io/project/5c4ec1443fcf8345e74aafb5/screen/5c6edc6a6d6dc44097589998